Date.php

```diff
@@ -4593,7 +4591,7 @@ function inEquivalentTimeZones($po_date1, $po_date2)
      * @access   public
      * @static
      */
-    function compare($od1, $od2)
+    static function compare($od1, $od2)
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.